src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java

```diff
@@ -146,6 +146,20 @@ public void whenListExistingEventTypesAreListed() throws NakadiException, Duplic
         assertThat(eventTypes, hasSize(2));
     }
 
+    @Test
+    public void whenRemoveThenDeleteFromDatabase() throws Exception {
+        EventType eventType = buildEventType();
+        String insertSQL = "INSERT INTO zn_data.event_type (et_name, et_event_type_object) VALUES (?, to_json(?::json))";
```

Comment:
Why not use `saveEventType(eventType)` here? Just to avoid touching of this code if the table structure will change in a future.